target/linux/ath79/generic/base-files/etc/board.d/01_leds

```diff
@@ -543,6 +543,15 @@ yuncore,a770)
 	ucidef_set_led_netdev "wan" "WAN" "green:wan" "eth1"
 	ucidef_set_led_switch "lan" "LAN" "green:lan" "switch0" "0x10"
 	;;
+yuncore,cpe830)
+	ucidef_set_led_netdev "wan" "WAN" "wan" "eth1"
+	ucidef_set_led_switch "lan" "LAN" "lan" "eth0"
```

Comment:
This won't work. If I'm not wrong the LED labels should be `"green:wan"`/`"green/lan"`.

Reply:
Please take care of this comment.

Reply:
I need to get back to original firmware to revisit the led codes, will do in a few